package.json

```diff
@@ -46,7 +46,9 @@
   },
   "dependencies": {
     "codem-isoboxer": "0.2.7",
-    "round10": "^1.0.3"
+    "imsc": "^1.0.0-beta.0",
+    "round10": "^1.0.3",
+    "sax": "^1.2.1"
```

Comment:
Is it necessary to add sax as a dependency here? Shouldn't imsc bring in this dependency? (Perhaps there is something I am missing ...)